Assets/HoloToolkit/Input/Scripts/InputManager.cs

```diff
@@ -727,6 +726,6 @@ public void RaiseDictationError(IInputSource source, uint sourceId, string dicta
         }
 
         #endregion // Dictation Events
-
+#endif
```

Comment:
I just opened the project in Unity. Are those flags added when you change the target platform? If they are then probably neither.